### PR TITLE
PDA messages now have a (F) link for ghosts

### LINKED ISSUE #13829: PDA messages seen by ghosts show no indication who is actually using the PDA
This is a pain for admins if we're trying to stop PDA spam or something because it requires opening up logs to see the ckey of the sender

### LINKED ISSUE #14087: PDA messages are still displayed while your life is slowly sapping away in crit

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Mostly so admins can go nuke the person who stole the laywer PDA to send WGW to everyone on station. Fixes #13829

Also fixes seeing PDA messages while in crit. Fixes #14087

### CODE REVIEW

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -769,16 +769,16 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	else
 		L = get(src, /mob/living/silicon)
 
-	if(L)
+	if(L && L.stat != UNCONSCIOUS)
```
Comment (phil235) [inline code/game/objects/items/devices/PDA/PDA.dm]: why not == CONSCIOUS ? Is there a case where we show the message to a dead mob?

Comment (KorPhaeron) [inline code/game/objects/items/devices/PDA/PDA.dm]: Yeah if I'm dead and waiting in my body to be cloned, there is no harm in me seeing PDA messages

**Comment by Supermichael777** [comment]
now seems too weak. maybe make it  -1,-1,3.  a decent area denial explosion with a some knockdown power, remember that explosion range counts the tile the weapon detonates in as tile 1 so that value will literally only cause knockdown if someone is standing on it.

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (tkdrg) [inline code/game/objects/items/devices/PDA/PDA.dm]: Well I mean, ghosts can see all PDA messages anyway, so it doesn't really matter.

Comment (KorPhaeron) [inline code/game/objects/items/devices/PDA/PDA.dm]: You're not a ghost if you're in your body though

Comment (tkdrg) [inline code/game/objects/items/devices/PDA/PDA.dm]: Yeah I just meant that it's information they can access either way, so it's not that big of a deal.

Comment (KorPhaeron) [inline code/game/objects/items/devices/PDA/PDA.dm]: My bad

**Pull request merged**